### PR TITLE
Feature/Pixel density and scale

### PR DESCRIPTION
**Pull request by ValeriiMalkov**
Implements [#2319](https://github.com/smartdevicelink/sdl_core/issues/2319)

This PR is **ready** for review.

### Testing Plan
The manual testing was done.

### Summary
Adding parameters to systemCapabilities according to a proposal.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by IGapchuk** [comment]
NOTE: feature branch has been rebased onto current develop

**Comment by IGapchuk** [comment]
@kshala-ford Please, be informed that feature branch has been rebased onto current develop. Feel free to approve it, or suggest required changes.

File: capabilities/rc_capabilities.js [hunk 1]
```diff
@@ -141,11 +141,10 @@ SDL.remoteControlCapability =
         fanSpeedAvailable: true,
         moduleName: 'primary_climate',
         ventilationMode: ['UPPER', 'LOWER', 'BOTH', 'NONE'],
-        ventilationModeAvailable: true,
-        climateEnableAvailable: true
+        ventilationModeAvailable: true
     }],
     radioControlCapabilities: [{
-        availableHdChannelsAvailable: true,
+        availableHDsAvailable: true,
```
Comment (kshala-ford) [inline capabilities/rc_capabilities.js]: It looks like some remote control code made it into this PR. Please remove code not related to pixel density and scale.

Comment (ValeriiMalkov) [reply capabilities/rc_capabilities.js]: @kshala-ford Fixed and updated

**Comment by AStasiuk** [comment]
@kshala-ford other reviewers were removed.

**Comment by AStasiuk** [comment]
@theresalech, this PR is already reviewed and approved by Ford (by @kshala-ford) and ready for Livio review.

File: capabilities/systemCapabilities.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                
+            },
+            {
+                protocol:  "WEBM",
+                codec: "VP9"
+                
+            }
+        ],
+        hapticSpatialDataSupported: true,
+        diagonalScreenSize: 22,
```
Comment (JackLivio) [inline capabilities/systemCapabilities.js]: 22 is quite a large screen size. Is there a specific reason for choosing this value?

Comment (ValeriiMalkov) [reply capabilities/systemCapabilities.js]: @JackLivio This is a some random value just for testing. If it make sens which value will be presented, we can change it.

Comment (JackLivio) [reply capabilities/systemCapabilities.js]: Can you please update it to 8?

Comment (JackLivio) [inline capabilities/systemCapabilities.js]:
```suggestion
        diagonalScreenSize: 8,
```

Comment (IGapchuk) [reply capabilities/systemCapabilities.js]: @JackLivio We have applied you commit suggestion. You can see it in new commit fd7e36c0f30f415a87b858a8434e6765baa6b8df

**Pull request merged**